payjoin/src/receive/mod.rs

```diff
@@ -345,26 +346,46 @@ impl UncheckedProposal {
     }
 
     /// The Sender's Original PSBT
-    pub fn extract_tx_to_schedule_broadcast(&self) -> bitcoin::Transaction {
-        self.psbt.clone().extract_tx()
+    pub fn psbt_transaction(&self) -> bitcoin::Transaction { self.psbt.clone().extract_tx() }
```

Comment:
This name change seems unrelated to the receiver check this PR is about and I'd prefer it wasn't made. extract_tx (for the specific api reason) tries to convey why the api is available, and extract lets you know it's calculating rather than returning something from memory.

Updating this would also merit a change to the documentation at the front matter of this file